src/plugins/explore/public/components/query_panel/components/footer/language_reference/ppl_language.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { EuiLink, EuiText } from '@elastic/eui';
+import { IDataPluginServices } from 'src/plugins/data/public';
+import { LanguageReference } from '../../../../../../../data/public';
+import { useOpenSearchDashboards } from '../../../../../../../opensearch_dashboards_react/public';
+
+const PPLReference = () => {
+  const opensearchDashboards = useOpenSearchDashboards<IDataPluginServices>();
+  const pplDocs = opensearchDashboards.services.docLinks?.links.noDocumentation.ppl.base;
+  const limitationDocs =
+    opensearchDashboards.services.docLinks?.links.noDocumentation.sqlPplLimitation.base;
+  const pplFullName = (
+    <FormattedMessage id="queryEnhancements.queryBar.pplFullLanguageName" defaultMessage="PPL" />
+  );
+  const limitationsLink = (
+    <FormattedMessage id="queryEnhancements.queryBar.pplLimitationDoc" defaultMessage="here" />
+  );
+
+  return (
+    <>
+      <EuiText size="s">
+        <p>
+          <FormattedMessage
+            id="queryEnhancements.queryBar.pplSyntaxOptionsDescription"
+            defaultMessage="Piped Processing Language ({pplDocsLink}) is a query language that focuses on processing data in a sequential, step-by-step manner. OpenSearch SQL/PPL language limitations can be found {limitationDocsLink}."
```

Comment:
I think you got this from the previous implementation right? If so it is out of scope, but this translation is problematic as it assumes that "OpenSearch SQL/PPL language limitations can be found" + "here" in that order is grammatically correct in all languages

Reply:
actually i don't think it requires that order, translators can still do something like

`See {limitationDocsLink} for OpenSearch SQL/PPL language limitations` (in another language)